lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -147,6 +147,8 @@ To manage the web server with systemd, you can use a unit configuration file lik
 
   [Service]
   Type=simple
+  User=sri
+  Group=sri
```

Comment:
I would just leave out the group, or at least call it something like `users`.

Reply:
I usually want to ensure a process does not remain in the `root` group, but the docs do say: "If no group is set, the default group of the user is used." So leaving out the group is reasonable.